service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -32,15 +52,23 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
             }
         }
 
+        ServiceBinding<?> primaryServiceBinding = serviceBindings[0];
         // Bind the service info for the first one passed in
-        binder.bind(ServiceInfo.class).toProvider(new ServiceInfoProvider(serviceBindings[0].serviceInterface()));
+        binder.bind(ServiceInfo.class).toProvider(
+                new ServiceInfoProvider(
+                        primaryServiceBinding.serviceInterface(),
+                        Arrays
+                                .stream(serviceBindings)
+                                .map(ServiceBinding::serviceInterface)
+                                .toArray(Class[]::new)
+                ));
```

Comment:
IIUC, if `MetricsService` made into `ServiceInfo` then `ServiceGateway` would have duplicates for all `MetricsService`'s ACLs... unless `MetricsService` is set to `lotacable=false`? :thinking: 

I will review that.

OTOH, even if not in `ServiceInfo`, `MetricsService` can be used by infra since monitoring tools and alike know the URI of anything that's deployed and know the paths of `MetricService` (no need for ACL's to be shout out).